src/js/components/CheckBox/checkbox.stories.js

```diff
@@ -194,6 +194,59 @@ class CheckBoxInsideButton extends Component {
   }
 }
 
+class CheckBoxWithStickyDiv extends Component {
+  state = {
+    checked: [],
+    checkboxes: Array(8)
+      .fill()
+      .map((_, i) => `item ${i + 1}`),
+  };
+
+  onCheck = (event, value) => {
+    const { checked } = this.state;
+    if (event.target.checked) {
+      checked.push(value);
+      this.setState({ checked });
+    } else {
+      this.setState({ checked: checked.filter(item => item !== value) });
+    }
+  };
+
+  render() {
+    const { checked, checkboxes } = this.state;
+    return (
+      <Grommet theme={grommet}>
+        <Box pad="large" align="center">
+          <Box
+            height="120px"
+            width="xsmall"
+            overflow="auto"
+            style={{
+              position: 'relative',
+              display: 'block',
+            }}
+          >
+            <Box
+              background={{ color: 'neutral-1' }}
+              style={{ position: 'sticky', top: 0 }}
+            >
+              Click Me!
```

Comment:
I think `Click Me!` is a little confusing because the header isn't clickable. will it make more sense to change it to `Check & Scroll` or `Click & Scroll` for example?

Reply:
@ShimiSun I like Click & Scroll. Thank you for reviewing.